src/WebApi/Infrastructure/Repositories/GenericRepository.cs

```diff
@@ -20,21 +20,21 @@ protected GenericRepository(MyDbContext dbContext, IMapper mapper)
         _mapper = mapper ?? throw new ArgumentNullException(nameof(mapper));
     }
 
-    public async Task<TModel> Create(TModel model)
+    public async Task<TModel> Create(TModel entity)
```

Comment:
To me this feels like the previous `TModel model` made more sense. 🤔 

@TomasNiemczyk98 what if instead of changing the name here, we changed it in [the interface](https://github.com/3PillarGlobal-Czechia/interview-app-api/blob/main/src/WebApi/Application/Repositories/IGenericRepository.cs)?

That should get rid of the warning, and should keep our naming nice and sensible. ☺️ 

_If we decide to do this, we should do that for all methods in this class where we'd introduce `TModel entity`_

Reply:
That would be a much nicer solution 🙂